services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts

```diff
@@ -167,15 +172,28 @@ export default class LiveSyncerService {
         case ResponseTypes.NewHeadsReponse:
           const { number } = parsedData.result as NewPbftBlockHeaderResponse;
           const formattedNumber = parseInt(number, 16);
-          this.pbftsQueue.add(
-            QueueJobs.NEW_PBFT_BLOCKS,
-            {
-              pbftPeriod: formattedNumber,
-              type: SyncTypes.LIVE,
-            } as QueueData,
-            JobKeepAliveConfiguration
-          );
-          this.logger.debug(`Pushed ${formattedNumber} into PBFT sync queue`);
+          if (this.historicalSyncerService.isRunning) {
+            await this.queueCache.add({
+              name: QueueJobs.NEW_PBFT_BLOCKS,
+              data: {
+                pbftPeriod: formattedNumber,
+                type: SyncTypes.LIVE,
+              } as QueueData,
+              JobKeepAliveConfiguration,
```

Comment:
Seems like a duplicate block. We do these in both cases

Reply:
Yes, however, we do add the period to different places.